cmd/maya-apiserver/cstor-operator/start.go

```diff
@@ -63,7 +63,7 @@ func Start() error {
 	controller, err := NewControllerBuilder().
 		withKubeClient(kubeClient).
 		withOpenEBSClient(openebsClient).
-		withspcSynced(spcInformerFactory).
+		withcspcSynced(spcInformerFactory).
```

Comment:
Does this work if order of these methods are shuffled?

Reply:
yes.